Ansible/ansible_collections/jfrog/platform/roles/postgres/tasks/install.yml

```diff
@@ -31,6 +31,15 @@
   environment:
     LC_ALL: "{{ postgres_locale }}"
 
+- name: Grab the FIPS status
```

Comment:
Could you add `changed_when: False` to keep idempotency

Reply:
I don't follow? Could you elaborate?

Reply:
I don't understand? Could you elaborate?

Reply:
Without `changed_when: False` we always will get `changed: true` for this task. You can get more info - https://ansible-lint.readthedocs.io/en/latest/default_rules.html#no-changed-when

Reply:
Ahh OK, I get you now. Sure, I can add that.